packages/core/src/network.js

```diff
@@ -1,7 +1,7 @@
 import { request as makeRequest } from '@percy/client/utils';
 import logger from '@percy/logger';
 import mime from 'mime-types';
-import { DefaultMap, createResource, hostnameMatches, normalizeURL, waitFor } from './utils.js';
+import { DefaultMap, createResource, hostnameMatches, normalizeURL, waitFor, modifyURL } from './utils.js';
```

Comment:
```suggestion
import { DefaultMap, createResource, hostnameMatches, normalizeURL, waitFor, decodeEncodeUrl } from './utils.js';
```

Reply:
Addressed. Named it `decodeAndEncodeURLWithLogging`